CONTRIBUTING.md

```diff
@@ -1,36 +1,62 @@
 # Contributing
 
-**Thanks for giving back to the open source community!**
+**✨ First off, thank you for your support in our mission to make terminals
+glamorous! ✨**
 
-Here are a few guidelines that make it easier for maintainers to review your
-pull requests...
+Here are some best practices to contribute to Charm ૮ ˶ᵔ ᵕ ᵔ˶ ა
 
-## Contributing changes
+Charm projects are open to contributions. The team is in maintenance mode for
+most of our tools right now, so it may take some time to review your
+contributions. During this time we will be prioritizing bug fixes. Regardless,
+the team always loves to hear from the community and sincerely appreciates all of
+your support!
```

Comment:
```suggestion
Charm projects are open to contributions. We're a small team with varied
priorities, so it may take some time to review your contributions. During this
time we will be prioritizing bug fixes. Regardless,the team always loves to
hear from the community and sincerely appreciates all of your support!
```